utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,244 @@
+/*
+  * Copyright 2020 Esri.
+  * Licensed under the Apache License, Version 2.0 (the "License");
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  * http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  */
+  
+package com.esri.samples.display_utility_associations;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Label;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.*;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.UniqueValueRenderer;
+import com.esri.arcgisruntime.utilitynetworks.UtilityAssociation;
+import com.esri.arcgisruntime.utilitynetworks.UtilityAssociationType;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetwork;
+import com.esri.arcgisruntime.utilitynetworks.UtilityNetworkSource;
+
+public class DisplayUtilityAssociationsSample extends Application {
+  
+  private MapView mapView;
+  private GraphicsOverlay associationsOverlay;
+  private UtilityNetwork utilityNetwork;
+  
+  @Override
+  public void start(Stage stage) {
+    
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
```

Comment:
We should add this line for the CSS file to actually be used:

```suggestion
      Scene scene = new Scene(stackPane);
      scene.getStylesheets().add(getClass().getResource("/display_utility_associations/style.css").toExternalForm());
```